src/main/scala/io/iohk/ethereum/faucet/jsonrpc/FaucetJsonRpcController.scala

```diff
@@ -1,32 +1,35 @@
 package io.iohk.ethereum.faucet.jsonrpc
 
 import io.iohk.ethereum.faucet.jsonrpc.FaucetDomain._
-import io.iohk.ethereum.jsonrpc.server.controllers.JsonRpcControllerCommon
-import io.iohk.ethereum.jsonrpc.server.controllers.JsonRpcControllerCommon.JsonRpcConfig
+import io.iohk.ethereum.jsonrpc.server.controllers.JsonRpcBaseController
+import io.iohk.ethereum.jsonrpc.server.controllers.JsonRpcBaseController.JsonRpcConfig
 import io.iohk.ethereum.jsonrpc.{JsonRpcError, JsonRpcRequest, JsonRpcResponse}
-
-import scala.concurrent.Future
+import io.iohk.ethereum.utils.Logger
+import monix.eval.Task
 
 class FaucetJsonRpcController(
     faucetRpcService: FaucetRpcService,
     override val config: JsonRpcConfig
-) extends JsonRpcControllerCommon
-    with ApisBuilder {
+) extends ApisBuilder
+    with Logger
+    with JsonRpcBaseController {
+
+  import FaucetMethodsImplicits._
 
-  override def enabledApis: Seq[String] = config.apis :+ Apis.Rpc
+  override def enabledApis: Seq[String] = config.apis
 
-  override def apisHandleFns: Map[String, PartialFunction[JsonRpcRequest, Future[JsonRpcResponse]]] = Map(
+  override def apisHandleFns: Map[String, PartialFunction[JsonRpcRequest, Task[JsonRpcResponse]]] = Map(
     Apis.Faucet -> handleRequest
   )
 
-  def handleRequest: PartialFunction[JsonRpcRequest, Future[JsonRpcResponse]] = { case req =>
-    val notFoundFn: PartialFunction[JsonRpcRequest, Future[JsonRpcResponse]] = { case _ =>
-      Future.successful(errorResponse(req, JsonRpcError.MethodNotFound))
+  def handleRequest: PartialFunction[JsonRpcRequest, Task[JsonRpcResponse]] = { case req =>
+    val notFoundFn: PartialFunction[JsonRpcRequest, Task[JsonRpcResponse]] = { case _ =>
+      Task(errorResponse(req, JsonRpcError.MethodNotFound))
```

Comment:
`Task.now(errorResponse(req, JsonRpcError.MethodNotFound))`

Reply:
Done, I applied it in other cases too.